src/compressor/Compressor.h

```diff
@@ -42,6 +42,17 @@ class Compressor {
 #endif
     COMP_ALG_LAST	//the last value for range checks
   };
+
+  // The names in this array must match the above values:
+  constexpr static char const* const compression_algorithm_names[] = {
+	"none",
```

Comment:
please note our tab-width is 8. and the indent offset is 2. so this indent does not follow the style guide.

Reply:
It matches all the code around it...